commit 29fd87e48ead14e0c00146c1705d2d0e720746c4
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-07-11T22:48:58+00:00 (3 months ago)
    
    TRITON-580 urclient could include some more details about who's running the command
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 6956625..74c2065 100644
--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
         "tab": "0.1.0",
         "triton-netconfig": "1.4.0",
         "ufds": "1.6.0",
-        "urclient": "1.0.0",
+        "urclient": "1.2.1",
         "vasync": "1.6.3",
         "verror": "1.10.0",
         "wordwrap": "1.0.0",
